### PR TITLE
WOE 12 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/verdant_outrider.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Verdant Outrider
+ManaCost:2 G
+Types:Creature Human Knight
+PT:4/2
+A:AB$ Effect | Cost$ 1 G | StaticAbilities$ KWPump | SpellDescription$ CARDNAME can't be blocked by creatures with power 2 or less this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/verdant_outrider.txt:5]: ExileOnMoved

File: forge-gui/res/cardsfolder/upcoming/welcome_to_sweettooth.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Welcome to Sweettooth
+ManaCost:1 G
+Types:Enchantment Saga
+K:Saga:3:DBToken,DBFood,DBPutCounter
+SVar:DBToken:DB$ Token | TokenScript$ w_1_1_human | TokenAmount$ 1 | TokenOwner$ You | SpellDescription$ Create a 1/1 white Human creature token.
+SVar:DBFood:DB$ Token | TokenScript$ c_a_food_sac | SpellDescription$ Create a Food token.
+SVar:DBPutCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | CounterType$ P1P1 | CounterNum$ X |  SpellDescriptionPut X +1/+1 counters on target creature you control, where X is one plus the number of Foods you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/welcome_to_sweettooth.txt:7]: desc syntax

File: forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Witchstalker Frenzy
+ManaCost:3 R
+Types:Instant
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each creature that attacked this turn.
+SVar:X:Count$AttackersDeclared
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt:5]: this only counts your own

File: forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Yenna, Redtooth Regent
+ManaCost:2 G W
+Types:Legendary Creature Elf Noble
+PT:4/4
+A:AB$ Pump | Cost$ 2 T | ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield | TgtPrompt$ Choose target enchantment you control that doesn't have the same name as another permanent you control. | SorcerySpeed$ True | SubAbility$ DBCopy | SpellDescription$ Choose target enchantment you control that doesn't have the same name as another permanent you control.
+SVar:DBCopy:DB$ CopyPermanent | Defined$ Targeted | RememberTokens$ True | NonLegendary$ True | SubAbility$ DBUntap | StackDescription$ Create a token that's a copy of it, except it isn't legendary. If the token is an Aura, untap CARDNAME, then scry 2. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt:6]: extra pump shouldn't be needed?

File: forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+SVar:DBUntap:DB$ Untap | Defined$ Self | ConditionDefined$ Remembered | ConditionPresent$ Card.Aura | SubAbility$ DBScry
+SVar:DBScry:DB$ Scry | ScryNum$ 2 | ConditionDefined$ Remembered | ConditionPresent$ Card.Aura | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | ClearImprinted$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt:9]: nothing imprinted?

File: forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt [hunk 6]
```diff
@@ -2,10 +2,9 @@ Name:Yenna, Redtooth Regent
 ManaCost:2 G W
 Types:Legendary Creature Elf Noble
 PT:4/4
-A:AB$ Pump | Cost$ 2 T | ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield | TgtPrompt$ Choose target enchantment you control that doesn't have the same name as another permanent you control. | SorcerySpeed$ True | SubAbility$ DBCopy | SpellDescription$ Choose target enchantment you control that doesn't have the same name as another permanent you control.
-SVar:DBCopy:DB$ CopyPermanent | Defined$ Targeted | RememberTokens$ True | NonLegendary$ True | SubAbility$ DBUntap | StackDescription$ Create a token that's a copy of it, except it isn't legendary. If the token is an Aura, untap CARDNAME, then scry 2. Activate only as a sorcery.
+A:AB$ CopyPermanent | Cost$ 2 T | Defined$ Targeted | RememberTokens$ True | NonLegendary$ True | ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield | TgtPrompt$ Choose target enchantment you control that doesn't have the same name as another permanent you control. | SorcerySpeed$ True | SpellDescription$ Choose target enchantment you control that doesn't have the same name as another permanent you control. | SubAbility$ DBUntap
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt:5]: Change Defined to ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield

Comment (squee1968) [reply forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt:5]: But I tried this both of these ways and couldn't get it to work

Comment (squee1968) [reply forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt:5]: Did you play test this?

Comment (squee1968) [reply forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt:5]: Change Defined$ parameter to ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield

File: forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt (hunk 3 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt]: `SVar:X:Count$CreaturesAttackedThisTurn Creature` is not correct ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt]: Same problem

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt]: I guess Rowdy Research is scripted wrong as well. Any tips what the correct variable is here ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt]: `PlayerCountPlayers$AttackersDeclared`

**Review by tool4ever — APPROVED** [review]

**Pull request merged**